### PR TITLE
Fix store downloading

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

File: libraries/generate_lib_list.py [hunk 1]
```diff
@@ -33,7 +30,7 @@ def main():
     config = load_config()
     with tempfile.NamedTemporaryFile() as temp:
         print('Downloading')
-        download('ubuntu-core', temp.name, config, 'amd64')
+        download('ubuntu-core', 'edge', temp.name, config, 'amd64')
```
Comment (kyrofa) [inline libraries/generate_lib_list.py]: Ah, I like being explicit. Edge now though, eh? We were using stable.

File: snapcraft/storeapi/_download.py [hunk 2]
```diff
@@ -22,38 +22,53 @@
...
-    details_url = _STORE_DETAILS_URL.format(snap)
-    details = session.get(details_url).content.decode('utf-8')
-    details_json = json.loads(details)
-    download_url = details_json.get('download_url')
-    download_sha = details_json.get('download_sha512')
+    pkg = search_results['_embedded']['clickindex:package']
+    if len(pkg) != 1:
+        raise EnvironmentError(
+            'Unexpected store result {!r}'.format(search_results))
+    download_url = pkg[0].get('anon_download_url', pkg[0]['download_url'])
```
Comment (kyrofa) [inline snapcraft/storeapi/_download.py]: Can you explain the anon versus non-anon here?

Comment (sergiusens) [inline snapcraft/storeapi/_download.py]: @kyrofa it is a mess

**Comment by sergiusens** [comment]
@elopio http://162.213.35.179:8080/job/github-snapcraft-autopkgtest-cloud/499/console says finished success, can we consider it a success?

**Comment by sergiusens** [comment]
retest this please

**Pull request merged**